.github/ISSUE_TEMPLATE.MD

```diff
@@ -0,0 +1,43 @@
+<!-- This form is for reporting bugs and issues specific to the Sitewide Sales plugin. This is not a support portal. If you need technical support from a human being, please submit a ticket via the Customer Support Area instead: https://www.strangerstudios.com/wordpress-plugins/sitewide-sales/support/ -->
+
+<!-- If you are a developer who needs a new filter/hook raise a PR instead :) -->
+
+<!-- Please be as descriptive as possible; issues lacking the below details, or for any other reason than to report a bug, may be closed without action. -->
+
+## Prerequisites
+
+<!-- MARK COMPLETED ITEMS WITH AN [x] -->
+
+- [ ] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
+- [ ] The issue still exists against the latest `dev` branch of Sitewide Sales on Github
+- [ ] I have attempted to find the simplest possible steps to reproduce the issue
+
+## Steps to reproduce the issue
+
+<!-- We need to be able to reproduce the bug in order to fix it so please be descriptive! -->
+
+1.
+2.
+3.
+
+## Expected/actual behavior
+
+When I follow those steps, I see...
+
+I was expecting to see...
+
+## Isolating the problem
+
+<!-- MARK COMPLETED ITEMS WITH AN [x] -->
+
+- [ ] This bug happens with only the Sitewide Sales plugin active
```

Comment:
Not sure if we want to say "with only the Sitewide Sales plugin and relevant module plugins active" if an issue is related to a particular integration

Reply:
I just made this improvement. Thank you!